TinyConstraints/Classes/Constrainable.swift

```diff
@@ -27,7 +27,7 @@
     import UIKit
 #endif
 
-extension TinyView: Constrainable {
+@MainActor extension TinyView: Constrainable {
```

Comment:
Not needed as the compiler infers `@MainActor` isolation for all members declared on this extension since `Constrainable` is isolated to the `@MainActor`.

```suggestion
extension LayoutGuide: Constrainable {
```